### ISSUE TITLE
[IDEA] Listen for X-Plane packets over the internet

### ISSUE THREAD

**Issue by DaveBC** [foucdeg/x-plane-map-electron#15]
Would it be possible to allow the user to configure where the app listens for x-plane packets? Currently there has to be an instance of the map running within the same local network as X-Plane. If I was to open the 49003 port to the internet, it would be a nice feature for the app to display the aircraft from another network that isn't running the app. This would need the option to change the listening IP from 127.0.0.1 to some other global IP.

**Comment by foucdeg** [comment]
Hello Dave,
Thanks for your input. 

I checked the code I wrote, and according to Node.js documentation, my server listeners (both on the X-Plane side and the map client side) attempt to bind to their assigned port on all available IPs and network interfaces. I have not set the listening IP to 127.0.0.1 (if I did, local network wouldn't work). So this should already work. But I can't easily try it out unfortunately.

See [this line of code](https://github.com/foucdeg/x-plane-map-electron/blob/master/src/udp.js#L80) and [Node documentation for `socket.bind()`](https://nodejs.org/docs/latest-v8.x/api/dgram.html#dgram_socket_bind_port_address_callback)

I'm curious about your setup. How is your port 49003 open to the internet? If it's forwarded by the router to your computer, your computer should see it as intra-network traffic, as if X-Plane was running on the router.
